tests/e2e_regression/expected/spec.body

```diff
@@ -1109,6 +1151,10 @@ components:
           type: string
           description: The delegatee address.
           example: *staking_address_1
+        address:
```

Comment:
Here and in `DebondingDelegation`: I don't think that the `address` of a delegation is obviously the delegator address, as opposed to the delegatee address.

Let's change this to `delegator_address`; or alternatively (and IMO better), just `delegator`, and rename `validator_address` to `validator`. (Nobody uses the consensus API yet, we can change it freely)

Reply:
:+1:  Updated to `validator` and `delegator`.